test/WebJobs.Script.Tests/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

```diff
@@ -6,27 +6,27 @@
   "compilationOptions": {},
```

Comment:
Can you back this file out, run the test, and give us the output (as a comment)? I see a couple of concerning major version bumps there and want to see them all summarized.

Reply:
sure

Reply:
@brettsam

Reply:
Well... I'm not sure we can do any of this without a bigger review. That's a lot of major version bumps. 

- Microsoft.Identity.Client.Extensions.Msal.dll: 2.31.0.0/2.31.0.0 -> 4.56.0.0/4.56.0.0
- Microsoft.Win32.SystemEvents.dll: 4.0.2.0/4.700.19.56404 -> 6.0.0.0/6.0.21.52210
- System.Configuration.ConfigurationManager.dll: 4.0.3.0/4.700.19.56404 -> 6.0.0.0/6.0.21.52210
- System.Security.Cryptography.ProtectedData.dll: 4.0.5.0/4.700.19.56404 -> 6.0.0.0/6.0.21.52210
- System.Security.Permissions.dll: 4.0.3.0/4.700.19.56404 -> 6.0.0.0/6.0.21.52210
- System.Windows.Extensions.dll: 4.0.1.0/4.700.19.56404 -> 6.0.0.0/6.0.21.52210